test/specs/01.displayname.spec.js

```diff
@@ -0,0 +1,93 @@
+// Unit Test that are related to annotations should be here
+
+var includes = require(__dirname + '/../utils/ermrest-init.js').init();
+
+var server = includes.server;
+var ermRest = includes.ermRest;
+var ermrestUtils = includes.ermrestUtils;
+
+describe('Display Name: ', function () {
+    var catalog_id, schemaName, schema, catalog;
+
+    // This function should be present in all spec files. It will add sample database and configurations.
+    beforeAll(function (done) {
+        ermrestUtils.importData({
+            setup: require('../configuration/displayname.spec.conf.json'),
+            url: includes.url ,
+            authCookie : includes.authCookie
+        }).then(function(data) {
+            catalog_id = data.catalogId;
+            schemaName = data.schema.name;
+            console.log("Data imported with catalogId " + data.catalogId);
+            done();
+        }, function(err) {
+            console.log("Unable to import data");
+            console.dir(err);
+            done.fail();
+        });
+    });
+
+    // Test Cases:
+
+    it('schema should use its own heuristics.', function(done) {
```

Comment:
this is too generic 'schema should use its own heuristics'. if i'm looking in a test and see it failed i would not understand what failed from that message.